spec.md

```diff
@@ -1920,7 +1970,7 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 
 | Condition | gRPC Code | Description | Recovery Behavior |
 |-----------|-----------|-------------|-------------------|
-| Snapshot in use | 9 FAILED_PRECONDITION | Indicates that the snapshot corresponding to the specified `snapshot_id` could not be deleted because it is in use by another resource. | Caller SHOULD ensure that there are no other resources using the snapshot, and then retry with exponential back off. |
+| Snapshot in use | 9 FAILED_PRECONDITION | Indicates that the snapshot corresponding to the specified `snapshot_id` could not be deleted because it is in use by another resource or it is part of a group snapshot. | Caller SHOULD ensure that there are no other resources using the snapshot, and then retry with exponential back off. |
```

Comment:
```suggestion
| Snapshot in use | 9 FAILED_PRECONDITION | Indicates that the snapshot corresponding to the specified `snapshot_id` could not be deleted because it is in use by another resource or it is part of a group snapshot and CAN NOT be deleted stand alone. | Caller SHOULD ensure that there are no other resources using the snapshot, and then retry with exponential back off. |
```

Reply:
Done